_posts/2020-12-24-how-to-start-with-foreman

```diff
@@ -0,0 +1,399 @@
+and as much as possible. In order to start using some virtualization backend, a compute
+resources must be defined. Compute resource represents the hypervisor or cloud.
+
+Some providers are shipped with Foreman but more can be added by installing plugins.
+As of today, these are part of Foreman core:
+
+* Amazon EC2
+* Google Compute Engine
+* Libvirt
+* OpenStack
+* oVirt / RHV
+* VMware (with vSphere only)
+
+Some popular compute resource types added by plugins are: Azure, Digital Ocean,
+HyperV, Scaleway, Xen. If your provider is not listed here, you can
+write the new plugin.
+
+Let's say you start using Foreman with some existing VM infrastructure. The easiest
+way to start is connecting Foreman to your virtualization and import all VMs from
+there. First we need to create a new compute resource object.
+
+Navigate to *Infrastructure -> Compute resources* and click *Create Compute Resource*
+button. Give it some name and select a provider type. If you don't see any provider,
+you need to install some. While some providers are part of Foreman core, we
+ship them in separate RPM packages, so users can install only those, that are
+interesting in their environment. In order to see, what providers you have
+installed, navigate to *Administer -> About -> Available Providers*.
+
+<p float='left' align='center'>
+  <img src="/static/images/blog_images/2020-12-24-how-to-start-with-foreman/cr_providers.png" width="1000" />
+</p>
+
+Here's a table with package name for each core provider
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Virtualization</th>
+    <th>RPM package name</th>
+  </tr>
+  <tr>
+    <td>Amazon EC2</td>
+    <td>foreman-ec2</td>
+  </tr>
+  <tr>
+    <td>Google compute engine</td>
+    <td>foreman-gce</td>
+  </tr>
+  <tr>
+    <td>Libvirt</td>
+    <td>foreman-libvirt</td>
+  </tr>
+  <tr>
+    <td>oVirt / RHV</td>
+    <td>foreman-ovirt</td>
+  </tr>
+  <tr>
+    <td>VMware</td>
+    <td>foreman-vmware</td>
+  </tr>
+</table>
+
+After installing new provider, restart the Foreman web server by running
+
+```
+service foreman restart
+```
+
+Due to lack of packaging capacity, for Debian based systems, we bundle all core
+providers into the main Foreman package. For more information, check out [manual](https://theforeman.org/manuals/latest/#5.2ComputeResources)
+
+After you choose the provider in the compute resource form, some additional
+fields show up. This is the form specific to the selected provider. Some providers
+allow to test connection, which is required to load additional information.
+The test only verifies the connection can be established but does not necessarily
+verify, that credentials are correct or the account has all required permissions.
+
+<p float='left' align='center'>
+  <img src="/static/images/blog_images/2020-12-24-how-to-start-with-foreman/create_vmware.png" width="1000" />
+</p>
+
+Tabs Organizations and Locations can be ignored for now, just make sure at least
```

Comment:
I recall reading something that recommended to avoid using words such as just or simply in tutorials because it implies those things are simple which for a beginner isn't always true.

```suggestion
The Organizations and Locations tabs can be ignored for now, but make sure at least
```